### PR TITLE
My journal week 4

### PR DESCRIPTION
**Pull request by rosalsm**

### CODE REVIEW

**Comment by boshnivolo** [comment]
Great start to your journal! I'm going to have to ask Karlo for his beautiful code too haha :+1:

**Comment by karjac14** [comment]
Great tutorial on DOM. And nice to know that we are all learning Jquery! :+1:

**Comment by ironyarder77** [comment]
Very Nice Rosa...You seem more in tune with this everyday!

File: tutorial-week-4.md [hunk 1]
```diff
@@ -1,13 +1,61 @@
 
 
-Maybe today I can write something different.
+# DOM
```
Comment (kellymurray) [inline tutorial-week-4.md:3]: Definitely add some set up here... why are you talking about the DOM? Why is it important to understand? What exactly is this tutorial?

**Pull request merged**